deployments/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -89,19 +89,19 @@ data:
     - name: {{ .Values.nsm.serviceName | quote }}
       labels:
         app: "vl3-nse-{{ .Values.nsm.serviceName }}"
-{{- if .Values.cnns.nsr.addr }}
-        cnns/nsr.addr: {{ .Values.cnns.nsr.addr | quote }}
-        cnns/nsr.port: {{ .Values.cnns.nsr.port | quote }}
+{{- if .Values.wcm.nsr.addr }}
+        wcm/nsr.addr: {{ .Values.wcm.nsr.addr | quote }}
+        wcm/nsr.port: {{ .Values.wcm.nsr.port | quote }}
 {{- end }}
-      cnns:
+      wcm:
         name: {{ .Values.nsm.serviceName | quote }}
-        address: "{{ .Values.cnns.nsr.addr }}"
+        address: "{{ .Values.wcm.nsr.addr }}"
         connectivityDomain: "{{ .Values.nsm.serviceName }}-connectivity-domain"
       vl3:
-       ipam:
-          defaultPrefixPool: {{ .Values.cnns.ipam.defaultPrefixPool | quote }}
-          serverAddress: "ipam-{{ .Values.cnns.nsr.addr }}:50051"
-          prefixLength: 22
+       wcmd:
+          defaultPrefixPool: {{ .Values.wcm.wcmd.defaultPrefixPool | quote }}
+          serverAddress: "wcmd-{{ .Values.wcm.nsr.addr }}:50051"
```

Comment:
hmm... the value side here should actually be `{{ .Values.ipam.serverAddress }}` the hardcoding of the prefix was a hack probably because the scripts/vl3/vl3_interdomain.sh wasn't setting the serverAddress.